src/components/Home/Home.js

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+import { Link } from "react-router-dom";
+import './Home.css';
+
+const Home = (props) => {
+
+  return (
+    <main>
```

Comment:
eventually will contain duck + small intro/instructions
e.g. "Can I be your duck?" [yes]

Reply:
TBD whether this component is actually necessary in addition to `App`